### DISCUSSION TITLE
I want to have access to ViewData inside layout.liquid

### DISCUSSION THREAD

**Discussion by abdulrahmannady** [OrchardCMS/OrchardCore#12967] [Q&A]
![Screenshot_3](https://user-images.githubusercontent.com/77383973/208896352-85c55538-678a-4c31-91a5-80489fbf19c3.png)

as you can seed if I have an option on ViewData[Fmt.IsFluid] is true I want to render body inside container-fluid bootstrap class

**Comment by abdulrahmannady** [comment]
this could be don using HtmlContext.Items like this snippet
this on layout.liquid
![1](https://user-images.githubusercontent.com/77383973/208940830-b5b5e49b-920c-4692-93f8-8d20e80fba0f.png)

this on razor page class
![2](https://user-images.githubusercontent.com/77383973/208940844-b331776b-7635-4b90-ba63-65252f73643a.png)

**Reply by hishamco** [reply]
You are using `HtmlContext.Items` not `ViewData`

**Reply by ns8482e** [reply]
It's not ViewData - The selected answer is not related to the question being asked
